### PR TITLE
Add button to pop out plugins panel v2

### PR DESCRIPTION
**Pull request by arch1t3cht**
The draggable plugins panel is impossible to discover for new users. The added button pops out the plugins panel without opening it fully, so that users can realize that the panel exists and is draggable.

The issue described in #192 is fixed now.

### CODE REVIEW

**Comment by Setsugennoao** [comment]
The Zen of Python, by Tim Peters

...
There should be one-- and preferably only one --obvious way to do it.
Although that way may not be obvious at first unless you're Dutch.
...

As i said i don't see the point in having this button, which is also redundant.
I don't see how a person can't see the grabbable thingy at the right, really, it's pretty obvious.

If you really want to dumb it down i guess a better approach, other than say "read the docs" or smth, is having a semi-transparent tooltip or the like that stays until you drag it out and have a flag in the global config or registry, like a first-time open "tutorial"

**Comment by LightArrowsEXE** [comment]
tbh I don't actually dislike the idea of a "first-time tutorial" kinda bubble to teach new users where it is. Though I disagree that it's "obvious". If it were as obvious as you think, we wouldn't get people asking us how to do this stuff.

**Comment by Setsugennoao** [comment]
well, vspreview isn't as simple as 2021 anymore so i had this idea for long, and not just for the plugin tab

**Comment by arch1t3cht** [comment]
The Zen of Python applies to the programming language, not to user-facing programs written in it. Redundancy is not always a bad thing when it comes to user interfaces.

> I don't see how a person can't see the grabbable thingy at the right, really, it's pretty obvious.

I have not seen a single new user spot this panel themselves, and I remember at least five people *asking* where to find plugins or where the comp tool went, only to be surprised that you can drag out a panel from the right.

Sure, a first-time tour would work but would be massively more complex and has issues like needing to save state, which would be out of date as soon as a new item gets added to the tour.
